src/utils/cvExport.js

```
@@ -0,0 +1,674 @@
+    case 'pdf':
+      return await exportToPDF(cvData, template, customOptions);
+    case 'docx':
+    case 'word':
+      return await exportToWord(cvData, template, customOptions);
+    case 'html':
+      return await exportToHTML(cvData, template, customOptions);
+    case 'txt':
+    case 'text':
+      // Simple text download (existing functionality)
+      const blob = new Blob([cvText], { type: 'text/plain' });
+      const filename = `${cvData.name.replace(/\s+/g, '_')}_CV.txt`;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)